File: Programmers/Java/Lv.0/[Lv.0] 가위 바위 보/[Lv.0] 가위 바위 보_00.java
class Solution {
    public String solution(String rsp) {
        String answer = "";
        String[] str = rsp.split("");

        for (int i = 0; i < rsp.length(); i++) {
            if (str[i].equals("0")) {
                answer += "5"; // 0 -> 5
            } else if (str[i].equals("2")) {
                answer += "0"; // 2 -> 0
            } else if (str[i].equals("5")) {
                answer += "2"; // 5 -> 2
            } else {
                answer = answer;
            }
        }

        return answer;
    }
}

File: Programmers/Python/Lv.0/[Lv.0] 가위 바위 보/[Lv.0] 가위 바위 보_00.py
def solution(rsp):
    answer = ''

    for i in rsp:
        if i == '0':
            answer += '5' # 0 -> 5
        elif i == '2':
            answer += '0' # 2 -> 0
        else:
            answer += '2' # 5 -> 2

    return answer
